src/showcases/components/button/buttonTheming.component.tsx

```diff
@@ -1,33 +1,61 @@
 import React from 'react';
 import { StyleSheet, View } from 'react-native';
-import { Button, Icon, Layout, Spinner } from '@ui-kitten/components';
+import { Button, Icon, IconElement, Layout, Spinner } from '@ui-kitten/components';
 
-const StarIcon = (props) => (
-  <Icon {...props} name='star'/>
+const StarIcon = (props): IconElement => (
+  <Icon
+    {...props}
+    name='star'
+  />
 );
 
-const LoadingIndicator = (props) => (
+const LoadingIndicator = (props): React.ReactElement => (
```

Comment:
the same here, declaring props as `ImageProps` will let to get rid of lint disabling